src/libnpf/npf.c

```diff
@@ -704,7 +702,6 @@ npf_rule_insert(nl_config_t *ncf, nl_rule_t *parent, nl_rule_t *rl)
 		rlset = ncf->ncf_rules_list;
 	}
 	prop_array_add(rlset, rldict);
-	prop_object_release(rldict);
```

Comment:
@outscale-mgo: Why? `prop_array_add()` acquires an additional reference to the dictionary. Unfortunately, proplib has an awful API.

P.S. I intend to replace proplib with, most likely, FreeBSD's libnv. How would that affect you? We can discuss this in greater detail via email.

Reply:
Yes, but when rlset is destroy(with prop_array_free), proplib release each elements of rlset,
So I guess we don't need to release  rldict here.

I don't think replace proplib with another lib would affect Packetgraph a lot.
So I don't have any problem with that.